Flutter/system-requirements.md

```diff
@@ -35,53 +35,46 @@ See the links below for the recommended Flutter development environments
+        <td style="text-align:center">>=<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/23.2.5">23.2.5</a> <24.2.x(<a href="https://pub.dev/packages?q=publisher%3Asyncfusion.com&page=2">latest</a>)
         </td>
     </tr>
      <tr>
         <td style="text-align:center">
            <a href="https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.13.0-stable.zip">3.13.0</a>
         </td>
-        <td style="text-align:center">
-            <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/22.2.11">22.2.11</a> to <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/23.2.4">23.2.4</a>
+        <td style="text-align:center">>=<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/22.2.11">22.2.11</a> <<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/23.2.4">23.2.4</a>
         </td>
     </tr>
      <tr>
         <td style="text-align:center">
            <a href="https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.10.0-stable.zip">3.10.0</a>
         </td>
-        <td style="text-align:center">
-            <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/21.2.9">21.2.9</a> to <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/22.2.10">22.2.10</a>
+        <td style="text-align:center">>=<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/21.2.9">21.2.9</a> <<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/22.2.10">22.2.10</a>
         </td>
     </tr>
      <tr>
         <td style="text-align:center">
            <a href="https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.7.0-stable.zip">3.7.0</a>
         </td>
-        <td style="text-align:center">
-            <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.4.50">20.4.50</a> to <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/21.2.8">21.2.8</a>
+        <td style="text-align:center">>=<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.4.50">20.4.50</a> <<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/21.2.8">21.2.8</a>
         </td>
     </tr>
      <tr>
         <td style="text-align:center">
            <a href="https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.3.0-stable.zip">3.3.0</a>
         </td>
-        <td style="text-align:center">
-            <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.3.47">20.3.47</a> to <a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.4.49">20.4.49</a>
+        <td style="text-align:center">>=<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.3.47">20.3.47</a> <<a href="https://pub.dev/packages/syncfusion_flutter_charts/versions/20.4.49">20.4.49</a>
```

Comment:
@LokeshPalani  All tags are closed properly?

Reply:
Hi @LavanyaGowtham2021,

I have checked all the tags and they were closed properly. Additionally, I have ensured that the Syncfusion Compatible package version is correct.

**Screenshot:**

![image](https://github.com/syncfusion-content/flutter-docs/assets/105289202/1eed4f4c-b93c-42f9-bed0-9f2b293649e9)


Regards,
Lokesh P.